exporters/elasticsearch/src/es_log_recordable.cc

```diff
@@ -71,7 +77,7 @@ void ElasticSearchRecordable::SetTimestamp(
 
   // If built with with at least cpp 20 then use std::format
   // Otherwise use the old style to format the timestamp in UTC
-#if __cplusplus >= 202002L
+#if defined(__cpp_lib_format) && __cpp_lib_format >= 201907
```

Comment:
I assume this is related to: https://en.cppreference.com/w/cpp/feature_test#cpp_lib_format

Please add a link to this doc in a comment.

Reply:
Done